### PR TITLE
[fix](jni-reader) fix hudi jni scanner

### PR DESCRIPTION
**Pull request by suxiaogang223**
## Proposed changes

Move the hadoop-aliyun and aliyun-sdk-oss dependency from fe-core to fe-common

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: fe/fe-common/pom.xml [hunk 1]
```diff
@@ -132,6 +132,15 @@ under the License.
...
         </dependency>
+        <dependency>
+            <groupId>com.aliyun.oss</groupId>
+            <artifactId>aliyun-sdk-oss</artifactId>
+            <version>3.15.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aliyun</artifactId>
+        </dependency>
```
Comment (CalvinKirs) [inline fe/fe-common/pom.xml:143]: If the user does not use Alicloud OSS, can we remove this dependency without causing any exceptions? I'm unsure if there will be any issues.

Comment (suxiaogang223) [reply fe/fe-common/pom.xml]: Do you mean to package it together again, but to ask the user to add this package manually?

**Comment by suxiaogang223** [comment]
run buildall

File: fe/fe-common/pom.xml (hunk 1 above)
Comment (suxiaogang223) [reply fe/fe-common/pom.xml]: I think it should not affect functions that do not depend on oss

**Review by morningman — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**